### DISCUSSION TITLE
iOS black stripe on top of the system overlay after routing to another page

### DISCUSSION THREAD

**Discussion by fuz12** [nylo-core/nylo#160] [General] [answered]
Hello! We face this problem starting from 5.64.0 nylo_support version, so we override dependency to 5.63.0 version of nylo_support. The problem is that a black stripe appears on top of the system overlay after routing to another page. This happens when there are others pages on top of the first.
![image](https://github.com/user-attachments/assets/30eb7480-ad04-4f3d-98ab-207bf727ca2d)

**Comment by agordn52** [comment, marked as answer]
Hi @fuz12,

This is because we added the swipe-back gesture for IOS in [v5.64.0](https://github.com/nylo-core/support/releases/tag/v5.64.0).

There are two ways you could fix this.

1. Update your Theme style to use a white background
2. Set **isIos** to false using `pageTransitionSettings` when routing.
```dart
router.route(..., pageTransitionSettings: PageTransitionSettings(
    isIos: false
));

// update routeTo calls

routeTo(..., pageTransitionSettings: PageTransitionSettings(
    isIos: false
));
```

Hope that helps :)

**Reply by fuz12** [reply]
No matter what colors I try to change, the first method doesn't work, but the second one does

**Reply by agordn52** [reply]
Hi @fuz12, 

Try update your `lib/bootstrap/app.dart` file.

Add a `Container` as the parent widget like in the below example.

``` dart
@override
Widget build(BuildContext context) {
    List<AppTheme> appThemes = Nylo.getAppThemes();
    return Container( // add this
      color: Colors.white,
...
```
